idl/github.com/uber/cadence/shared.thrift

```diff
@@ -1230,10 +1230,20 @@ struct DescribeTaskListRequest {
   10: optional string domain
   20: optional TaskList taskList
   30: optional TaskListType taskListType
+  40: optional bool isAdmin
```

Comment:
Rather than isAdmin lets called this something like: detailed. The reason here is I want to keep all admin related terminology as isolated from the rest of the codebase as possible. 

Another name that I like is includeTaskListStatus. You are already referring to this collection of fields as status in your comment "status of tasklist's ackManager (readLevel, ackLevel, backlogCountHint and taskIDBlock)."